CONTRIBUTING.md

```diff
@@ -1,47 +1,224 @@
+
+The GitHub issue is sufficient for such discussions, and can be sufficient to get clarity on what you plan to do. Make sure you tag the Amplify Native team using @aws-amplify/amplify-native so we can help guide you.
+
+### Step 3: Fork the Repo
+
+First, create a fork of amplify-ios. Clone it, and make changes to this fork.
+
+```
+git clone git@github.com:GITHUBUSERNAME/aws-sdk-ios.git 
+# change your GITHUBUSERNAME to your github username before running this command.
+```
+
+### Step 4: Work your magic
+
+Work your magic. Here are some guidelines:
+
+- Coding style (abbreviated):
+    - In general, follow the style of the code around you
+    - 4 space indentation
+    - 120 characters wide
+    - Every change requires a new or updated unit test/integ test
+    - If you change customer facing APIs, make sure to update the documentation above the interface and include a reason for the breaking change in your PR comments
+    - Try to maintain a single feature/bugfix per pull request. It's okay to introduce a little bit of housekeeping changes along the way, but try to avoid conflating multiple features. Eventually all these are going to go into a single commit, so you can use that to frame your scope.
+
+### Step 5: Commit
+
+Create a commit with the proposed change changes:
+
+- Commit message should describe motivation. Think about your code reviewers and what information they need in order to understand what you did. If it's a big commit (hopefully not), try to provide some good entry points so it will be easier to follow.
+
+### Step 6: Pull Request
+
+- Push your changes to your GitHub fork
+- Submit a Pull Requests on the aws-sdk-ios repo to the `master` branch and add the Amplify Native team using @aws-amplify/amplify-native so we can approve/provide feedback.
+- The title of your PR must be descriptive to the specific change.
+- No period at the end of the title.
+- Pull Request message should indicate which issues are fixed: `fixes #<issue>` or `closes #<issue>`.
+- PR messaged should include shout out to collaborators.
+- If not obvious (i.e. from unit tests), describe how you verified that your change works.
+- If this PR includes breaking changes, they must be listed at the top of the changelog as described above in the Pull Request Checklist.
+- Discuss review comments and iterate until you get at least one “Approve”. When iterating, push new commits to the same branch. 
+- Usually all these are going to be squashed when you merge to master.
+- Make sure to update the PR title/description if things change. 
+- Rebase with master if the master branch has commits ahead of your fork.
+
+### Step 7: Merge
+Once your PR has been approved and tested, go ahead and merge it into `master`! Barring any unforeseen circumstances, your changes will be released in our next version. Yay!! 
```

Comment:
Contributors can't do this, only people with `write` permissions on the repo (i.e., us)

Reply:
updated